webpack.config.js

```diff
@@ -86,7 +86,7 @@ var config = {
                 ],
             },
             {
-                test: /\.scss$/,
+                test: /\.(css|scss)$/,
```

Comment:
`/\.s?css$/`
This also works.

Reply:
I'm good with what we have. The current version is much easier to read

Reply:
sounds good.